src/pages/StudyBuddyPage/components/CreateAd/CreateAd.tsx

```diff
@@ -0,0 +1,486 @@
+import { useState } from "react";
+import Button, { ButtonsEnum } from "../../../../shared/ui/Button/Button";
+import style from "./CreateAd.module.css";
+import ConfirmModal from "../../../../shared/ui/ConfirmModal/ConfirmModal";
+
+export default function CreateAd({ setIsActive }: { setIsActive: any }) {
```

Comment:
shouldn't be any, sounds like a function

Reply:
Fixed